src/apis/share.ts

```diff
@@ -49,7 +49,9 @@ export const createStorageItem = async (
 };
 
 //팀 자료 다운로드
-export const downloadStorageItem = async (storageId: number): Promise<Blob> => {
+export const getDownloadStorageItem = async (
```

Comment:
```suggestion
export const getTeamDocuments = async (
```

get이랑 download가 같은 의미인 것 같아서 getTeamDocuments 등으로 수정되면 좋을 것 같아요! 다른 의견 있으면 말씀해 주세요!

Reply:
앗 좋아요!! 그렇게 수정하겠습니당